packages/luna/src/components/Calendar/v2/WeekDays.tsx

```diff
@@ -0,0 +1,75 @@
+import { range } from "lodash";
+import React, { useMemo } from "react";
+import { DAYS_IN_WEEK } from "@/components/Calendar/v2/constants";
+import { Dayjs } from "dayjs";
+import cn from "classnames";
+import dayjs from "@/lib/dayjs";
+import { Typography } from "@/components/Typography";
+import { DayView } from "@/components/Calendar/v2/types";
+import { HOURS_IN_DAY } from "@litespace/sol/constants";
+
+export const WeekDays: React.FC<{
+  day: Dayjs;
+  DayView?: DayView;
+}> = ({ day, DayView }) => {
+  const week = useMemo(() => {
+    const weekStart = day.startOf("week");
+    return range(DAYS_IN_WEEK).map((day) => {
+      const dayStart = weekStart.add(day, "day").startOf("day");
+      const hours = range(HOURS_IN_DAY - 1).map((hour) =>
+        dayStart.add(hour, "hour")
+      );
+      return { day: dayStart, hours };
+    });
+  }, [day]);
+
+  return (
+    <>
+      {week.map(({ day, hours }, idx) => {
+        const today = day.isSame(dayjs(), "day");
+        const lastDay = week.length - 1 === idx;
+        return (
+          <div
+            className="tw-border-r tw-border-natural-300"
+            key={day.toISOString()}
+          >
+            <div
```

Comment:
This is the div that should take `tw-absolute`. This is the div that the day view should be positioned against.